### DISCUSSION TITLE
Hinweis gewünscht - "Es steht noch kein programmierter Mitspieler zur Verfügung"

### DISCUSSION THREAD

**Discussion by ElbPsi** [fabiankachlock/zwoo#256] [Ideas]
Hallo Team,

ich hätte gerne einen "Computer-Mitspieler". 
Solange es noch keinen gibt bitte einen Hinweis, dass ich nicht alleine spielen kann.
Grüße,
Elb

**Comment by fabiankachlock** [comment]
Hallo Elb,
vielen Dank für die Idee.

Das Thema technische Mitspieler wird zwar bereits intern vorbereitet, ist aber weder fachlich fertig geplant, noch wurde die Umsetzung zeitlich eingeplant. Auf diesem langen Weg bis zur Fertigstellung kann noch viel passieren, so steht zum Beispiel noch nicht fest, wie und an welcher Stelle in der Benutzeroberfläche technische Mitspieler später bearbeitet werden können.
Solange solche grundlegenden Entscheidungen noch nicht getroffen sind, haben wir uns dazu entschieden, nicht auf Features aufmerksam zu machen, die eventuell sogar noch ganz gestrichen werden können.

zwoo Team
